### ISSUE TITLE
Installation Completes with Errors.  Update Command Not Found

### ISSUE THREAD

**Issue by weaponplus** [sans-dfir/sift#99]
I've attempted to install SIFT on a clean Ubuntu 14.04 install a couple of times and it never completes properly.  Many of the packages get an Error 100.  When the installation completes, I try to perform an "update-sift", but I get a command not found error.  

My workstation is behind a proxy and I'm using CNTLM to authenticate.  I've added 8.8.8.8 as an additional DNS in order to prevent some of the DNS issues I've read about in this forum.  

My installation log is at:
https://gist.github.com/weaponplus/6441f1b8d76857260330

**Comment by ekristen** [comment]
According to your logs, your proxy authentication is failing and thus packages are failing to install. 

https://gist.github.com/weaponplus/6441f1b8d76857260330#file-gistfile1-txt-L254-L296

**Comment by weaponplus** [comment]
What's so strange is that I'm able to browse websites while all of the errors are occurring.  I'm not sure why authentication seems to fail at the same point each time I try the install.  I just ran it again, after switching proxies and I'm getting the same output.

**Comment by ekristen** [comment]
apt generally uses a different proxy setup then your basic browsers

**Comment by PoorBillionaire** [comment]
Also be sure to explicitly preserve your http_proxy and https_proxy environment variables when invoking sudo:

http://stackoverflow.com/questions/8633461/how-to-keep-environment-variables-when-using-sudo

**Comment by madsumm** [comment]
Hi all,
not sure this helps, but it worked for me. 
I noticed this when I set up 2 VM. One could work and the others don't. I realise a small difference in the updater from Ubuntu.

I went into "Software & Updates" and click on Updates.
UNCHECKED Pre-released updates AND Unsupported Updates.
Run the software updater again from Ubuntu to clear out the old.

After which, run the SIFT wget.
This time round it worked perfectly with zero errors.

Of course, ensure all package managers and updaters have completed prior to start SIFT Wget.
It seems some packages are conflicting?
Reds
Mathew
